bin/shpec

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/sh
```

Comment:
shellcheck may have been overreaching with this one. We haven't needed a shebang so far, and the intention is for shpec to be POSIX-compliant and runnable in whatever shell.

However, I would love to get a second opinion from someone with more experience on POSIX...